core/build.gradle

```diff
@@ -67,4 +69,9 @@ dependencies {
     testImplementation "org.assertj:assertj-core:$assertj_version"
     testAnnotationProcessor "com.google.dagger:dagger-compiler:$dagger_version"
     testAnnotationProcessor "com.google.dagger:dagger-android-processor:$dagger_version"
+    compile "androidx.core:core-ktx:+"
+    implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
+}
+repositories {
```

Comment:
We don't add `repositories` in the `module` `build.gradle` file unless it is extremely necessary. I believe in this case it isn't. The project's `build.gradle` should already be including this repository.